### PR TITLE
Fixes #141 : Fixes PHP Fatal error: No base path provided when run php public.index.php

### LINKED ISSUE #141: fresh install, run  php public/index.php got PHP Fatal error:  No base path provided
Steps to reproduce:
1. Download apigility 1.4.0 from https://apigility.org/download
2. Create new API
3. Run command from `php public/index.php`
```
$ php public/index.php 
PHP Fatal error:  Uncaught Zend\View\Exception\RuntimeException: No base path provided in /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Helper/BasePath.php:38
Stack trace:
#0 [internal function]: Zend\View\Helper\BasePath->__invoke('/img/favicon.pn...')
#1 /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(394): call_user_func_array(Object(Zend\View\Helper\BasePath), Array)
#2 /Users/samsonasik/www/apigility/module/Application/view/layout/layout.phtml(10): Zend\View\Renderer\PhpRenderer->__call('basePath', Array)
#3 /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(502): include('/Users/samsonas...')
#4 /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/View.php(207): Zend\View\Renderer\PhpRenderer->render()
#5 /Users/samsonasik/www/apigility/vendor/zendframework/zend-mvc/src/View/Http/DefaultRenderingStrategy.php(105): Zend\View\View->render(Object(Zend\View\Model\ViewModel))
#6 /Users/samsonasik/www/apigility/vendor/zend in /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Helper/BasePath.php on line 38

Fatal error: Uncaught Zend\View\Exception\RuntimeException: No base path provided in /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Helper/BasePath.php on line 38

Zend\View\Exception\RuntimeException: No base path provided in /Users/samsonasik/www/apigility/vendor/zendframework/zend-view/src/Helper/BasePath.php on line 38

Call Stack:
    0.0009     358312   1. {main}() /Users/samsonasik/www/apigility/public/index.php:0
    2.9370    4623032   2. Zend\Mvc\Application->run() /Users/samsonasik/www/apigility/public/index.php:43
    2.9374    4625112   3. Zend\Mvc\Application->completeRequest() /Users/samsonasik/www/apigility/vendor/zendframework/zend-mvc/src/Application.php:326
    2.9374    4625112   4. Zend\EventManager\EventManager->triggerEvent() /Users/samsonasik/www/apigility/vendor/zendframework/zend-mvc/src/Application.php:369
    2.9374    4625112   5. Zend\EventManager\EventManager->triggerListeners() /Users/samsonasik/www/apigility/vendor/zendframework/zend-eventmanager/src/EventManager.php:143
    2.9382    4710656   6. Zend\Mvc\View\Http\DefaultRenderingStrategy->render() /Users/samsonasik/www/apigility/vendor/zendframework/zend-eventmanager/src/EventManager.php:271
    3.1214    4954008   7. Zend\EventManager\EventManager->triggerEvent() /Users/samsonasik/www/apigility/vendor/zendframework/zend-mvc/src/View/Http/DefaultRenderingStrategy.php:123
    3.1214    4954008   8. Zend\EventManager\EventManager->triggerListeners() /Users/samsonasik/www/apigility/vendor/zendframework/zend-eventmanager/src/EventManager.php:143
    3.1215    4955232   9. Zend\Mvc\View\Http\DefaultRenderingStrategy->render() /Users/samsonasik/www/apigility/vendor/zendframework/zend-eventmanager/src/EventManager.php:271
```

### PR DESCRIPTION
**Pull request by samsonasik**
fixed by add :
```
        "zendframework/zend-mvc-console": "^1.1",
```
to require section of composer.json

### CODE REVIEW

**Comment by samsonasik** [comment]
and add : 
```
'Zend\Mvc\Console',
```
to module lists.

**Comment by webimpress** [comment]
`zend-mvc-console` is compatible only with ZF3 (zend-mvc v3). It shouldn't be then in require section because it should be possible to use apigility with ZF2 as well as ZF3. Maybe it could be added to `suggest` section (?)

**Comment by samsonasik** [comment]
@webimpress done ;)

File: config/modules.config.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
     'Zend\Filter',
     'Zend\Hydrator',
     'Zend\InputFilter',
+    //'Zend\Mvc\Console', // activate it if you use zend-mvc v3
```
Comment (weierophinney) [inline config/modules.config.php]: zend-mvc-console has integration with zend-component-installer; there's no reason to add a commented-out entry, as installation of the component will add the entry.

**Comment by weierophinney** [comment]
I'm actually going to decline this patch.

We no longer recommend the MVC &lt;-&gt; Console tooling, and [even document as such in the zend-mvc migration guide](https://docs.zendframework.com/zend-mvc/migration/to-v3-0/#migrate-your-console-tooling); adding a suggestion for it runs counter to our own recommendations.

**Pull request closed without merging**